src/renderers.js

```diff
@@ -292,18 +292,26 @@ exports.defaultHocRenderer = defaultHocRenderer;
 function defaultComponentRenderer ({section, renderer, imports, typeRenderer = renderType}) {
 	const props = section.members.instance.filter(member => !member.kind);
 	const funcs = section.members.instance.filter(member => member.kind === 'function');
-
 	const omits = section.tags.reduce((res, tag) => tag.title === 'omit' ? res.concat(tag.description) : res, []);
 	const propsBase = calcPropsBaseName({imports, section});
 	const propsInterfaceName = `${section.name}Props`;
 	const propsInterface = renderInterface(propsInterfaceName, props, propsBase, typeRenderer, imports, omits);
+	let staticMembersDescriptionArray = [], staticMembersDescription = '';
 
 	imports.add({
 		module: 'react',
 		name: 'React',
 		all: true
 	});
 
+	if (section.members.static.length > 0) {
```

Comment:
please check the user story to read the available information about static members. Based on that info I created the staticMembersDescription